test/helpers/helpers.go

```diff
@@ -456,6 +475,50 @@ func RestartMarketplace(client test.FrameworkClient, namespace string) error {
 	return nil
 }
 
+// ScaleDownMarketplace scales the marketplace deployment down to zero
+func ScaleDownMarketplace(client test.FrameworkClient, namespace string) error {
```

Comment:
Nit: I suggest using this in `RestartMarketplace()` otherwise it is never going to get updated.